### ISSUE TITLE
Allow users to edit / revoke permissions?

### ISSUE THREAD

**Issue by suj87** [xfrocks/bdApi#136]
In XF 1 - regular users can edit or revoke permissions for clients (and see a list of clients). This does not seem to be there in XF 2.1.4 and bd API 2.0.1 Beta 4. Is this going to be implemented?

**Comment by daohoangson** [comment]
Yes, this feature is planned. User definitely should be able to that.

**Comment by suj87** [comment]
Ok great, thanks 🙂

**Comment by suj87** [comment]
Any ETA on when this will be implemented? Doesn't seem to be available in XF 2.2.2, API 2.2.0 Beta 1. Thanks

**Comment by suj87** [comment]
Monthly bump - any ETA ? :)

**Comment by suj87** [comment]
@daohoangson Any ETA? Thanks

**Comment by suj87** [comment]
Bump :)

**Comment by suj87** [comment]
Bump @daohoangson
